wolfssh/ssh.h

```diff
@@ -239,6 +239,7 @@ enum WS_UserAuthResults {
     WOLFSSH_USERAUTH_INVALID_AUTHTYPE,
     WOLFSSH_USERAUTH_INVALID_USER,
     WOLFSSH_USERAUTH_INVALID_PASSWORD,
+    WOLFSSH_USERAUTH_PASSWORD_RETRYOUT,
```

Comment:
Please rename the new authentication code "WOLFSSH_USERAUTH_REJECTED".

Reply:
Changed it with WOLFSSH_USERAUTH_REJECTED